### DISCUSSION TITLE
Using the dev/1 on Android

### DISCUSSION THREAD

**Discussion by Unchakable** [eclipse-milo/milo#1331] [General]
I am developing a small Android mobile application for reading data from a controller. Since JAXB is not supported on Android, I can only use the dev/1 branch. But the Gradle project collector is used on android and I can't register it:

<parent>
        <groupId>org.eclipse.milo</groupId>
        <artifactId>milo-examples</artifactId>
        <version>1.0.0-SNAPSHOT</version>
 </parent>

I'm connecting dependencies, but I can't access the dev/1 functionality.

dependencies {
    implementation(libs.appcompat)
    implementation(libs.material)
    implementation(libs.activity)
    implementation(libs.constraintlayout)
    testImplementation(libs.junit)
    androidTestImplementation(libs.ext.junit)
    androidTestImplementation(libs.espresso.core)
    implementation(libs.sdk.client)
    implementation(libs.eclipse.sdk.server)
    implementation(libs.stack.core)
    implementation(libs.stack.client)
    implementation(libs.sdk.core)
    implementation(libs.dictionary.reader)
}

I tried using plugin org.springframework.boot, but it causes critical errors during the build. Can I access the dev/1 branch in another way?

Thanks!

**Comment by kevinherron** [comment]
The `1.0.0-SNAPSHOT` artifacts aren't published anywhere, you need to check out the Milo project, switch to the `dev/1.0` branch, and then run `mvn clean install` (using JDK 17) to install the artifacts into your local Maven repository.

**Comment by kevinherron** [comment]
I've added a CI job to publish the `dev/1.0` branch to the Sonatype OSS snapshots repository.

If you add the snapshots repo as a source you can get it from there now as well.

```xml
<repository>
  <id>maven-snapshots</id>
  <url>http://oss.sonatype.org/content/repositories/snapshots</url>
  <releases>
    <enabled>false</enabled>
  </releases>
  <snapshots>
    <enabled>true</enabled>
  </snapshots>
</repository>
```
